modules/fastcv/test/test_moments.cpp

```diff
@@ -30,14 +29,25 @@ TEST_P(fcv_momentsTest, accuracy)
 
 	cv::Moments m = cv::fastcv::moments(src, binaryImage);
 
-    int len_m = sizeof(m)/sizeof(m.m00);
-    EXPECT_FALSE(len_m != 24);
+    cv::Scalar mean_val, stdDev;
+    float mean_val_fcv = m.m00/(srcSize.width * srcSize.height);
+    if(binaryImage)
+    {
+        cv::Mat src_binary(srcSize, CV_8UC1);
+        cv::compare( src, 0, src_binary, cv::CMP_NE );
+        mean_val = cv::mean(src_binary);
+        mean_val_fcv *= 255;
+    }
+    else
+        mean_val = cv::mean(src);
+
+    EXPECT_NEAR(mean_val[0], mean_val_fcv, 2);
```

Comment:
These tests do not converge on my phone:
```
[ RUN      ] fcv_momentsTest.accuracy/1, where GetParam() = (false, 640x480, 4)
/home/savuor/code/qc-fastcv-ext/opencv_contrib/modules/fastcv/test/test_moments.cpp:44: Failure
The difference between mean_val[0] and mean_val_fcv is 2642411.6411946616, which exceeds 2, where
mean_val[0] evaluates to -2660369.5142415366,
mean_val_fcv evaluates to -17957.873046875, and
2 evaluates to 2.
[  FAILED  ] fcv_momentsTest.accuracy/1, where GetParam() = (false, 640x480, 4) (24 ms)
[ RUN      ] fcv_momentsTest.accuracy/4, where GetParam() = (false, 1280x720, 4)
/home/savuor/code/qc-fastcv-ext/opencv_contrib/modules/fastcv/test/test_moments.cpp:44: Failure
The difference between mean_val[0] and mean_val_fcv is 246997.88133572048, which exceeds 2, where
mean_val[0] evaluates to 26830.587414279515,
mean_val_fcv evaluates to 273828.46875, and
2 evaluates to 2.
[  FAILED  ] fcv_momentsTest.accuracy/4, where GetParam() = (false, 1280x720, 4) (72 ms)
[ RUN      ] fcv_momentsTest.accuracy/7, where GetParam() = (false, 1920x1080, 4)
/home/savuor/code/qc-fastcv-ext/opencv_contrib/modules/fastcv/test/test_moments.cpp:44: Failure
The difference between mean_val[0] and mean_val_fcv is 530242.66986786271, which exceeds 2, where
mean_val[0] evaluates to -360915.18549286266,
mean_val_fcv evaluates to 169327.484375, and
2 evaluates to 2.
[  FAILED  ] fcv_momentsTest.accuracy/7, where GetParam() = (false, 1920x1080, 4) (161 ms)

[ RUN      ] fcv_momentsTest.accuracy/10, where GetParam() = (false, 127x61, 4)
/home/savuor/code/qc-fastcv-ext/opencv_contrib/modules/fastcv/test/test_moments.cpp:44: Failure
The difference between mean_val[0] and mean_val_fcv is 5544038.4969020262, which exceeds 2, where
mean_val[0] evaluates to -11985612.996902026,
mean_val_fcv evaluates to -6441574.5, and
2 evaluates to 2.
[  FAILED  ] fcv_momentsTest.accuracy/10, where GetParam() = (false, 127x61, 4) (1 ms)
```

Reply:
same as above